src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -72,46 +78,19 @@ class ElectronicDeliveryForm extends React.Component {
           <legend>Contact Information</legend>
           <h3>Contact Information</h3>
 
-          <div className={`nypl-text-field ${errorClass.name}`}>
-            <label htmlFor="patron-name" id="patron-name-label">First and last name
```

Comment:
I believe they are email, chapter title, start and end page.
This is from the API's wiki:
https://github.com/NYPL/hold-request-service/wiki/API-v0.1